.github/workflows/build-main-branches.yml

```diff
@@ -6,6 +6,7 @@ on:
         branches:
             - main
             - 2.x
+            - jakarta
```

Comment:
Do we need a new branch?

Reply:
It was to have the result of the transformation: https://github.com/smallrye/smallrye-reactive-messaging/tree/jakarta. 

How do you want to handle it here? For other SR projects, I keep a `jakarta` branch, to do the RC releases. When I need to update from `main`, I reset the branch to main and apply the transformations again.